rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -24,14 +24,18 @@ use windows::Win32::{
     },
     Networking::WinSock::{AF_INET, AF_INET6},
 };
+use wintun::Adapter;
+
+// Not sure how this and `TUNNEL_NAME` differ
+const ADAPTER_NAME: &str = "Firezone";
```

Comment:
I had originally extracted this so I could use it when deleting the adapter, but deleting is not necessary if I just join the worker thread.